kentico-mvc-environment/azuredeploy.json

```diff
@@ -100,7 +103,7 @@
       "minLength": 1,
       "defaultValue": "[resourceGroup().location]",
       "metadata": {
-        "description": "Location for all resources."
+        "description": "Represents the location of the specified resource group. This setting is required by Azure. Please do not change the default value in any way; changing the default value could cause issues during resource deployment."
```

Comment:
Why is that?

Reply:
I wasn't able to find any reference point where the possible location values are listed. It is safer for the user of the template to leave the location as it is to default to the resource groups location and not encounter any problems with deployment when guessing the correct values.

Reply:
The resource location and resourceGroup().location are not related - resources can be deployed into any location the are available and may not actually be available in the resourceGroup's location.  What's the specific reason location is a concern?

Reply:
As I mentioned, I did not find any complete list of valid string values that can be used as an input in the location parameter. If we could reference it in the documentation, the recommendation to stick with the default value would be redundant. When using the template in the Azure portal and choosing, or creating a resource group, the location can be nicely chosen from a dropdown which is a better user experience. And relying on this value in the location parameter is for preventing possible deployment issues with a wrong input in the location parameter.

Reply:
Removed the part of the description stating not to change the value.